lib/policies/rate-limit/rate-limit.js

```diff
@@ -11,6 +11,11 @@ module.exports = (params) => {
       }
     };
   }
+
+  params.handler = (req, res) => {
+    res.egError(429);
```

Comment:
make sense, 
could you please add error message, as I understand result will be just {code:429} ?  

not sure if we need a header, up to you 
https://github.com/nfriedly/express-rate-limit/blob/master/lib/express-rate-limit.js#L25

Reply:
Result will be:
```
{
  "code": 429,
  "error": "Too Many Requests"
}
```

When message is empty, it gets default http status message from `statuses` lib.